checklist.html

```diff
@@ -41,21 +48,23 @@ <h2 id="toc_title" class="toc-title">Categories</h2>
 			<input name="aria-navigation-role" id="navigation-role" aria-describedby="navigation-role-description" type="checkbox">
 		</label>
 
-		<p id="navigation-role-description" class="description">Contains navigational links.</p>
+		<p id="navigation-role-description" class="description">Contains links to navigate to different pages of a website, screens of an application, or a sections within a single document.</p>
 
 		<!-- main -->
 		<label for="main-role" class="checkbox"><code>&lt;main role="main"&gt;</code>
 			<input name="aria-main-role" id="main-role" aria-describedby="main-role-description" type="checkbox">
 		</label>
 
-		<p id="main-role-description" class="description">Focal content of document. Use only once.</p>
+		<p id="main-role-description" class="description">Wraps the focal content of document. Use only once.</p>
 
 		<!-- article -->
 		<label for="article-role" class="checkbox"><code>&lt;article role="article"&gt;</code>
 			<input name="aria-article-role" id="article-role" aria-describedby="article-role-description" type="checkbox">
 		</label>
 
-		<p id="article-role-description" class="description">Represents an independent item of content. Use only once on outermost element of this type.</p>
+		<p id="article-role-description" class="description">
+      Represents an independent item of content. There may be many &lt;article&gt;s in a single document. &lt;article&gt;s are not considered landmarks, but screen readers may still surface <code>article</code>s when navigating by regions or landmarks in a document.
```

Comment:
Let's be consistent when writing about code or elements of code. 

If we're discussing an HTML element or attribute name, wrap with `<code>` for formatting. I personally don't care for adding `&lt;` or `&gt;` as these are awkwardly spoken aloud by screen readers.

Reply:
agreed.  removed all instances of &lt; and &gt; (outside of once instance where there were multiple elements being referenced).  updating instances of elements not being wrapped in `code` elements to be.